keycloakapi/src/main/kotlin/de/klg71/keycloakmigration/keycloakapi/KeycloakClientInit.kt

```diff
@@ -18,17 +18,18 @@ import feign.jackson.JacksonEncoder
  * @param clientId id of the client to use for the login of the user
  */
 fun initKeycloakClient(baseUrl: String, adminUser: String, adminPassword: String, realm: String,
```

Comment:
i would suggest we make this method internal (for tests only) and add a new publicMethod where we add the parameter `tokenRefreshOffset` (name is arguable) that calls the internal method. Therefore we can still mock the `TokenHolder` but don't have to make the class public

Reply:
done. made it so that we keep previous "internal" notations